Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -907,7 +962,13 @@ define([
 
             ++tileProvider._usedDrawCommands;
 
-            command.debugShowBoundingVolume = (tile === tileProvider._debug.boundingSphereTile);
+            if (tile === tileProvider._debug.boundingSphereTile) {
+                if (defined(surfaceTile.orientedBoundingBox)) {
+                    createDebugOrientedBoundingBox(surfaceTile.orientedBoundingBox, Color.RED).update(context, frameState, commandList);
```

Comment:
Here and below leak the debug primitive on each frame.  We should keep a reference to it so we can call `destroy`.  Stop by and I can explain more.